assets.php

```diff
@@ -39,7 +35,7 @@ function action_enqueue_block_editor_assets() {
  *
  * @return array An array of dependencies for this asset.
```

Comment:
I can't tell based on the diff, perhaps this is missing a @param comment?

Reply:
Nope, there aren't any params on this function.